config/samples/backends/README.md

```diff
@@ -20,29 +20,54 @@ Currently available samples are:
 - NFS
 - LVM using iSCSI
 - LVM using NVMe-TCP
+- HPE 3PAR iSCSI
+- HPE 3PAR FC
+- NetApp ONTAP iSCSI
+- NetApp ONTAP FC
+- NetApp ONTAP NFS
+- Pure Storage iSCSI
+- Pure Storage FC
+- Pure Storage NVMe-TCP
```

Comment:
TCP is not availble until Bobcat. You can use NVMe-RoCE instead which is supported in Antelope

Reply:
Thanks! I incorrectly used the doc reference from master.